src/client/app/admin/authentication/inactive.component.ts

```diff
@@ -1,16 +1,17 @@
 import { Component } from '@angular/core';
-import { User } from '../user.class';
+
+import { User } from '../user';
+
 import { AuthenticationService } from './authentication.service';
 
 @Component({
-	templateUrl: './inactive.component.html'
+	templateUrl: './inactive.component.html',
+	providers: [ User ]
 })
-
 export class InactiveComponent {
-	user: User;
-
 	constructor(
-		private auth: AuthenticationService
+		private auth: AuthenticationService,
+		private user: User
```

Comment:
What exactly is the behavior here? Are we injecting a local instance of User into the user property and then overwriting it with the user from the authentication service? Is there a benefit to this approach versus just locally declaring the user variable?

Also, our convention is to mark anything referenced by the template as public. So, user should be public (which it was previously).